### PR TITLE
Add install/update instructions for PHIVE to the README

### LINKED ISSUE #3838: Document PhiVE release channel

### PR DESCRIPTION
**Pull request by SpacePossum**
closes #3838

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -85,6 +85,15 @@ Globally (homebrew)
...
     $ brew install php-cs-fixer
 
+Locally (PHIVE)
+~~~~~~~~~~~~~~~~~~~
+
+Install `PHIVE <https://phar.io>`_ and issue the following command:
+
+.. code-block:: bash
+
+    $ phive install php-cs-fixer # use `--global` for global install, not recommended
```
Comment (keradus) [inline README.rst]: I would drop `, not recommended`. It's opinionated, and even while I agree with it, we don't put that for other ways of installation, don't we ?
Here, i brings the confusion that installing sth with PHIVE globally is not recommended, it's not true.
We simply don't recommend to install dev-tools globally (as their versions depends on project), regardless which way of installation was chosen

Comment (keradus) [reply README.rst]: i would keep it

Comment (keradus) [reply README.rst]: i would keep it

File: src/Console/Command/ReadmeCommand.php [hunk 2]
```diff
@@ -131,6 +131,15 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
     $ brew install php-cs-fixer
 
+Locally (PHIVE)
+~~~~~~~~~~~~~~~~~~~
```
Comment (kubawerlos) [inline src/Console/Command/ReadmeCommand.php]: Why so long? Other underline-ish lines are with the length of the text above.

Comment (SpacePossum) [reply src/Console/Command/ReadmeCommand.php]: I'll admit, the lengthy `~` is a result of copying another (sub)header, I'll update soon, thanks for the sharp review :+1:   :)

File: README.rst [hunk 3]
```diff
@@ -124,6 +133,13 @@ You can update ``php-cs-fixer`` through this command:
 
     $ brew upgrade php-cs-fixer
 
+Locally (PHIVE)
+~~~~~~~~~~~~~~~~~~~
```
Comment (kubawerlos) [inline README.rst]: No so fast, this one is also too long ;)

Comment (SpacePossum) [reply README.rst]: oh no, I really really _really_ thought I had this covered, f**k me, this is a lesson learned on my "I wished wasn't needed "-list :(
aaarrggh I'm sorry man, you called me out on it before and still I messed it up, f**k me :(

Comment (keradus) [reply README.rst]: no worry. instead of fixing that, let us run rst / md linters on CI :D

**Pull request merged**